lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2974,6 +2974,65 @@ class DropEnumType(schema._CreateDropBase):
     __visit_name__ = "drop_enum_type"
 
 
+_server_side_id = util.counter()
```

Comment:
**mike bayer** (zzzeek) wrote:

we have no need for this in base.py.  this is entirely a pg8000 thing.  DBAPIs should be implementing server side cursors.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2776